### PR TITLE
Generation of pact files works

### PR DESCRIPTION
**Pull request by corinnekrych**
@bartoszmajsak this PR to our poc branch does:
* use chrome headless instad of phantomJS
* import pact (which internally import es6 promise) before zone.js to remove this exception:
```
Unhandled promise rejection Error: Zone.js has detected that ZoneAwarePromise(window|global
```
* check-in generated file so you see it was generated ;)

### CODE REVIEW

**Comment by bartoszmajsak** [comment]
Awesome!

giphy 25

File: config/karma.conf.js [hunk 1]
```diff
@@ -94,25 +98,39 @@ module.exports = function (config) {
...
+          ' --remote-debugging-port=9222'
+        ]
       }
     },
-    phantomjsLauncher: {
-      // Have phantomjs exit if a ResourceError is encountered
-      // (useful if karma exits without killing phantom)
-      exitOnResourceError: true
-    },
+    // browsers: ['PhantomJS_custom'],
```
Comment (bartoszmajsak) [inline config/karma.conf.js]: Not sure who we do it in this project, but if that's dead code maybe we can simply remove it?

File: config/spec-bundle.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ Error.stackTraceLimit = Infinity;
 // https://github.com/angular/angular-cli/issues/1675
 require('intl');
 require('intl/locale-data/jsonp/en');
-
+require('pact-web');
```
Comment (bartoszmajsak) [inline config/spec-bundle.js]: Short comment why or link to the PR emphasizing why the order is important could help?

File: package.json [hunk 3]
```diff
@@ -103,6 +103,7 @@
     "@angular/upgrade": "4.0.1",
     "@angularclass/hmr": "1.2.2",
     "@angularclass/hmr-loader": "3.0.2",
+    "@pact-foundation/pact-node": "^4.12.0",
```
Comment (bartoszmajsak) [inline package.json]: Is it needed? I thought it comes transitively from `pact`?

File: src/app/space/wizard/services/forge-wizard.api.spec.ts [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+import { step_1_1_response } from './forge-wizard.api.mock';
+import {HttpModule} from "@angular/http";
+import {Fabric8ForgeService} from "./fabric8-forge.service";
+import {LoggerFactory} from "../common/logger";
+import {AuthenticationService} from "ngx-login-client";
+import {ApiLocatorService} from "../../../shared/api-locator.service";
+
+describe('Pact Server:', () => {
+
+  let provider = Pact({ consumer: 'Karma Jasmine', provider: 'Hello', web: true });
```
Comment (bartoszmajsak) [inline src/app/space/wizard/services/forge-wizard.api.spec.ts]: Let's name consumer and provider properly :)

File: src/polyfills.browser.ts [hunk 5]
```diff
@@ -26,7 +26,7 @@ import 'core-js/es6/typed';
 import 'core-js/es6/reflect';
 // see issue https://github.com/AngularClass/angular2-webpack-starter/issues/709
 // import 'core-js/es6/promise';
-
+import 'pact-web';
```
Comment (bartoszmajsak) [inline src/polyfills.browser.ts]: Sigh :) So in this world of amazing web development you have more than 1 place where you define your dependencies? (also `spec-bundle.js`).

**Comment by fabric8cd** [comment]
@corinnekrych fabric8/fabric8-ui:SNAPSHOT-PR-1938-1 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1938-fabric8-ui.badger.fabric8.io

**Pull request merged**